tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/remote/RemoteInfinispanSessionDataStoreTest.java

```diff
@@ -69,6 +70,12 @@ public void teardown() throws Exception
         __testSupport.teardown();
     }
 
+    @AfterAll
```

Comment:
We're (re)setting a static `__testSupport` before every method call. Shouldn't `__testSupport` be assigned just once either statically or `@BeforeAll` and then we call `__testSupport.setup()` and `__testSupport.teardown()` `@BeforeEach` and `@AfterEach`?

Reply:
good point! this will make the tests running faster